aries_cloudagent/transport/outbound/tests/test_http_transport.py

```diff
@@ -66,7 +67,7 @@ async def send_message(transport, payload, endpoint, api_key):
             send_message(transport, "{}", endpoint=server_addr, api_key=api_key), 5.0
         )
         assert self.message_results == [{}]
-        assert self.headers.get("x-api-key") == api_key
+        assert api_key and compare_digest(self.headers.get("x-api-key"), api_key)
```

Comment:
I don't think there's any need to use this comparison method in the test

Reply:
okay, I'll remove it